server/src/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -1111,6 +1111,20 @@ public boolean canOperateOnVolume(Volume volume) {
     }
 
     @Override
+    public void cleanupSnapshotsByVolume(Long volumeId) {
+        List<SnapshotVO> volSnapShots = _snapshotDao.listByVolumeId(volumeId);
+        for(SnapshotVO snapshot: volSnapShots) {
+            SnapshotInfo info = snapshotFactory.getSnapshot(snapshot.getId(), DataStoreRole.Primary);
+            try {
+                snapshotSrv.deleteSnapshot(info);
+            } catch(CloudRuntimeException e) {
+                String msg = "Cleanup of Snapshot with uuid " + snapshot.getUuid() + " in primary storage is failed. Ignoring";
```

Comment:
This local variable is only used once.  Please consider collapsing into lint 1122.  Also, please add the message from the exception to the message to provide greater detail for debugging efforts.